README.md

```diff
@@ -18,14 +18,55 @@ The CoseSign1, CoseSign1.Abstractions, and CoseSign1.Certicates libraries provid
 CoseSignTool, CoseHandler, and the CoseSign1 libraries are the Microsoft solution for signing SBOMs and, we believe, the most powerful and convenient solution currently on the market.
 
 ## How do I get started?
-First, download the latest release from GitHub. There will be a fully signed version on NuGet.org soon, but this is [just a pre-release](#state-of-the-project), so there's only the open source version available for now.
+
+### Using as an executable CLI
+Downloadable versions are available in GitHub [releases](https://github.com/microsoft/CoseSignTool/releases) of this repository. Separate page lists the features and how to use them: [CoseSignTool.md](./docs/CoseSignTool.md).
+
+#### Linux
+Download and extract the folder with the compiled binaries, then make `CoseSignTool` available on the `$PATH`.
+
+```bash
+# download and uzip the release
+mkdir -p ~/cosesigntool
+curl -L https://github.com/microsoft/CoseSignTool/releases/download/v1.2.8/CoseSignTool-Linux-release.zip -o ~/cosesigntool/release.zip

```

Comment:
```suggestion
curl -L https://github.com/microsoft/CoseSignTool/releases/latest/download/CoseSignTool-Linux-release.zip -o ~/cosesigntool/release.zip
```

will age somewhat better (although it will still break if the file name changes)

Reply:
done